server/config/default.json

```diff
@@ -39,9 +39,10 @@
       "pass": "CATALOGUE_APPRENTISSAGE_SMTP_AUTH_PASS"
     }
   },
-  "oldCatalogue": {
-    "endpoint": "CATALOGUE_APPRENTISSAGE_CATALOGUE_API_ENDPOINT",
-    "apiKey": "CATALOGUE_APPRENTISSAGE_CATALOGUE_API_KEY"
+  "tableCorrespondance": {
+    "endpoint": "https://tables-correspondances.apprentissage.beta.gouv.fr",
```

Comment:
Can you add a variable for the endpoint, so that etablissement created in local or recette are not created in production ?